velox/exec/TableScan.cpp

```diff
@@ -256,6 +256,7 @@ void TableScan::preload(std::shared_ptr<connector::ConnectorSplit> split) {
        ctx = operatorCtx_->createConnectorQueryCtx(
            split->connectorId, planNodeId(), connectorPool_),
        task = operatorCtx_->task(),
+       &pendingDynamicFilters = pendingDynamicFilters_,
```

Comment:
Do we need a reference here? We pass everything else by value.

Reply:
Thanks for catching this, we should pass by value here